library/sha1.c

```diff
@@ -71,8 +71,15 @@
 }
 #endif
 
+#define MBEDTLS_SHA1_VALIDATE_RET(cond)                             \
+    MBEDTLS_VALIDATE_RET( MBEDTLS_ERR_SHA1_BAD_INPUT_DATA, cond )
```

Comment:
The order of these macro arguments is wrong.

Reply:
Fixed.